deeplabcut/pose_cfg.yaml

```diff
@@ -26,6 +26,16 @@ global_scale: 0.8
 dataset_type: imgaug
 batch_size: 1
 
+# Probability with which the augmenters will be applied to input images
+apply_prob: 0.5
+
+# Resize images prior to augmentation
+pre_resize: []  # Specify [width, height] if pre-resizing is desired
+
+# Smart, on-the-fly image cropping, replacing deeplabcut.cropimagesandlabels
+crop_size: [400, 400]  # width, height
```

Comment:
Is `crop_size` also set as a passable list to the create_training_dataset fxns?

Reply:
Good point! I'll add it :)